app/views/admin/layouts/partials/subnav-settings.blade.php

```diff
@@ -7,4 +7,7 @@
     <li class="{{ setActive('winners', 3, 'active') }}">{{ link_to_route('admin.winner.index', 'Winners') }}</li>
     <li class="{{ setActive('email', 2, 'active') }}">{{ link_to_route('emails', 'Emails') }}</li>
   </ul>
+  {{ Form::open(array('route' => 'general.clear-cache')) }}
+    {{ Form::submit('Clear Cache', ['class' => 'btn btn-default -hover-red']) }}
```

Comment:
Fenders and Benders Designin'

![tumblr_ntvnwpeckc1uyqh1fo1_400](https://cloud.githubusercontent.com/assets/1700409/9589044/c4eb4caa-4ff9-11e5-8d4f-fe6684d32b38.gif)